astropy/utils/tests/test_yaml.py

```diff
@@ -0,0 +1,99 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module tests some of the methods related to YAML serialization.
+
+Requires `pyyaml <http://pyyaml.org/>`_ to be installed.
+"""
+import numpy as np
+
+from ...coordinates import SkyCoord, EarthLocation, Angle, Longitude, Latitude
+from ... import units as u
+from ...time import Time
+
+from ...tests.helper import pytest
+
+try:
+    from ..yaml import load, dump
+    HAS_YAML = True
+except ImportError:
+    HAS_YAML = False
+
+pytestmark = pytest.mark.skipif('not HAS_YAML')
+
+
+@pytest.mark.parametrize('c', [u.m, u.m / u.s, u.hPa, u.dimensionless_unscaled])
+def test_unit(c):
+    cy = load(dump(c))
+    assert c == cy
```

Comment:
Actually, this should be true for all the units that are defined, i.e., all but `u.m / u.s`.